styles.css

```diff
@@ -0,0 +1,463 @@
+.header {
+  height: 330px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+  background-image: image("images/header-image.jpg");
+  overflow: hidden;
+}
+
+.header-img {
+  width: 100%;
+  margin-right: 0;
+}
+h1 {
+  color: #44BE2F;
+  padding-top: 26%;
+  padding-left: 3rem;
+  font-weight: 500;
+  font-style: italic;
+}
+
+.header-logo {
+  text-align: left;
+  position: absolute;
+  top: 2rem;
+  left: 5rem;
+}
+
+.header-checkout {
+  text-align: left;
+  position: absolute;
+  top: 10px;
+  right: 2%;
+  font-family: monospace;
+  font-weight: 700;
+  font-size: 1em;
+  text-decoration: none;
+  color: #fff;
+}
+
+.header-checkout a:link {
+  text-decoration: none;
+  padding: 4px 8px 4px 8px;
+  background-color: #9bc344;
+  border-radius: 1px;
+  border: none;
+  width: 240px;
```

Comment:
Still want to be careful to use px for sizing or padding, fontsize etc. about the only place px is acceptable is defining border line width.

Reply:
changed.